### PR TITLE
friston and grossberg citations

### PR DESCRIPTION
**Pull request by lucywowen**
Frist00a
Gros88

### CODE REVIEW

File: memlab.bib [hunk 1]
```diff
@@ -110,6 +110,27 @@ @string{vn
...
+	Date-Added = {2020-06-16 18:07:07 +0000},
+	Date-Modified = {2020-06-16 18:07:07 +0000},
+	Journal = {Neural networks},
+	Number = {1},
+	Pages = {17--61},
+	Title = {Nonlinear neural networks: Principles, mechanisms, and architectures.},
+	Volume = {1},
+	Year = {1988}}
+
+@article{Frist00a,
```
Comment (jeremymanning) [inline memlab.bib:124]: Should be Fris00a (or Fris00 if there isn't another Friston article from 2000)

File: memlab.bib [hunk 2]
```diff
@@ -10216,6 +10237,7 @@ @article{FaraMcCl91
...
 	Author = {Lyon, Don R and Gunzelmann, Glenn and Gluck, Kevin A},
+	Bdsk-File-1 = {YnBsaXN0MDDSAQIDBFxyZWxhdGl2ZVBhdGhZYWxpYXNEYXRhXxAeMjAwOC1seW9uX2d1bnplbG1hbm5fZ2x1Y2sucGRmTxEB4gAAAAAB4gACAAANbm91dmVhdU1hY1BybwAAAAAAAAAAAAAAAAAAyedG5EgrAAADEilwHjIwMDgtbHlvbl9ndW56ZWxtYW5uX2dsdWNrLnBkZgAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAIjVzJTH5YAAAAAAAAAAAAAQACAAAJIAAAAAAAAAAAAAAAAAAAAAhtYWdlbGxhbgAQAAgAAMnnfyQAAAARAAgAAMlMxKgAAAABABADEilwAAjFJwAIe3AAAJigAAIAVW5vdXZlYXVNYWNQcm86VXNlcnM6AHJvYmVydHNla3VsZXI6AERyb3Bib3g6AG1hZ2VsbGFuOgAyMDA4LWx5b25fZ3VuemVsbWFubl9nbHVjay5wZGYAAA4APgAeADIAMAAwADgALQBsAHkAbwBuAF8AZwB1AG4AegBlAGwAbQBhAG4AbgBfAGcAbAB1AGMAawAuAHAAZABmAA8AHAANAG4AbwB1AHYAZQBhAHUATQBhAGMAUAByAG8AEgBDVXNlcnMvcm9iZXJ0c2VrdWxlci9Ecm9wYm94L21hZ2VsbGFuLzIwMDgtbHlvbl9ndW56ZWxtYW5uX2dsdWNrLnBkZgAAEwABLwAAFQACABT//wAAAAgADQAaACQARQAAAAAAAAIBAAAAAAAAAAUAAAAAAAAAAAAAAAAAAAIr},
```
Comment (jeremymanning) [inline memlab.bib:10240]: not sure what this is...

File: memlab.bib (hunk 1 above)
Comment (lucywowen) [reply memlab.bib:124]: there are 3 from that year, published under 'the labile brain' and in the same journal

File: memlab.bib (hunk 2 above)
Comment (lucywowen) [reply memlab.bib:10240]: i have no idea what this is

Comment (lucywowen) [reply memlab.bib:10240]: i just removed it

**Pull request merged**